src/profile-logic/stack-timing.ts

```diff
@@ -87,6 +87,7 @@ export type StackTiming = {
   sameWidthsStart: number[];
   sameWidthsEnd: number[];
   callNode: IndexIntoCallNodeTable[];
+  argumentValues?: number[];
```

Comment:
nit: could you add a comment explaining that this is used for JS execution tracing argument values?